Sources/ExyteChat/ChatView/MessageView/MessageView.swift

```diff
@@ -132,35 +132,34 @@ struct MessageView: View {
 
     @ViewBuilder
     func bubbleView(_ message: Message) -> some View {
-        ZStack(alignment: message.user.isCurrentUser ? .topLeading : .topTrailing) {
-            VStack(alignment: .leading, spacing: 0) {
-                if !message.attachments.isEmpty {
-                    attachmentsView(message)
-                }
-                
-                if !message.text.isEmpty {
-                    textWithTimeView(message)
-                        .font(Font(font))
-                }
-                
-                if let recording = message.recording {
-                    VStack(alignment: .trailing, spacing: 8) {
-                        recordingView(recording)
-                        messageTimeView()
-                            .padding(.bottom, 8)
-                            .padding(.trailing, 12)
-                    }
-                }
+        VStack(alignment: .leading, spacing: 0) {
+            if !message.attachments.isEmpty {
+                attachmentsView(message)
             }
-            .bubbleBackground(message, theme: theme)
-            .applyIf(isDisplayingMessageMenu) {
-                $0.frameGetter($viewModel.messageFrame)
+            
+            if !message.text.isEmpty {
+                textWithTimeView(message)
+                    .font(Font(font))
             }
             
+            if let recording = message.recording {
+                VStack(alignment: .trailing, spacing: 8) {
+                    recordingView(recording)
+                    messageTimeView()
+                        .padding(.bottom, 8)
+                        .padding(.trailing, 12)
+                }
+            }
+        }
+        .bubbleBackground(message, theme: theme)
+        .overlay(alignment: message.user.isCurrentUser ? .topLeading : .topTrailing) {
```

Comment:
I don't really get it: overlay doesn't "extend" the layout, meaning it won't make the message cell bigger to fit the reaction, which "sticks out" from the message, no?

Reply:
On the screenshot "Before" there is "Sss" text. The scroll view in the bottom position, I didn't scroll it up. So the bubble is go further than the ZStack frame. I think it happened because of the fixedSize vertical parameter insize of the bubble view. If you text several "Sss" in row, you can have a better view
<img width="229" alt="image" src="https://github.com/user-attachments/assets/36ac57d8-f056-4f47-b2dc-72188189dbea" />